pkg/asset/agent/mirror/cabundle_test.go

```diff
@@ -0,0 +1,73 @@
+package mirror
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/mock"
+)
+
+func TestCaBundle_LoadedFromDisk(t *testing.T) {
+
+	cases := []struct {
+		name          string
+		data          string
+		fetchError    error
+		expectedFound bool
+		expectedError bool
+	}{
+		{
+			name: "valid-config-file",
+			data: `
+-----BEGIN CERTIFICATE-----
+MIIDvTCCAqWgAwIBAgICEAA....
+----- END CERTIFICATE -----`,
+			expectedFound: true,
+		},
+		{
+			name:          "empty",
+			data:          "",
+			expectedFound: true,
```

Comment:
As we started to do in other [similar tests](https://github.com/openshift/installer/blob/98bfce1602686f503fbcadf54b7011bddca0f129/pkg/asset/agent/manifests/agentpullsecret_test.go#L87), for the sake of clarity (and to avoid ignoring unexpected errors) I'd suggest to check explicitly the error message string for those tests where an error is expected

Reply:
+1 to check the error message.